http/NgapS3Credentials.h

```diff
@@ -24,30 +24,32 @@
 #ifndef HYRAX_GIT_S3CREDENTIALS_H
 #define HYRAX_GIT_S3CREDENTIALS_H
 
-#include "AccessCredentials.h"
 #include <string>
 
+#include "AccessCredentials.h"
```

Comment:
This reordering of includes makes no sense to me. I thought we always included the language stuff before our own includes.